File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.go
func getRow(rowIndex int) []int {
    row := []int{1}
    for i := 0; i < rowIndex; i++ {
        next := make([]int, len(row)+1)
        for j := 1; j < len(row); j++ {
            next[j] = row[j-1] + row[j]
        }
        next[0], next[len(next)-1] = 1, 1
        row = next
    }
    return row
}

File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        row = [1]
        for i in range(rowIndex):
            new_row = []
            for i in range(1, len(row)):
                num = row[i-1] + row[i]
                new_row.append(num)
            row = [1] + new_row + [1]
        return row    
                    